src/flows/pipes/Notification/endpoints/HTTP/index.ts

```diff
@@ -55,11 +55,15 @@ export default register => {
         task_data
           |> schema["fieldsAsCols"]()
               |> set(key: "_notebook_link", value: "${window.location.href}")
+          |> filter(fn: ${measurement})
```

Comment:
monitor.check() will fail if we have >1 measurement in the piped data.

Even once the query builder forces a single measurement, it will still be possible (in the raw flux editor) to have >1 measurement. Therefore, this requirement in the Alerts Panel will not go away.